src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

Combine this selectors and all below into one, like this:

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```